core/primitives/src/views.rs

```diff
@@ -415,6 +416,7 @@ pub struct BlockHeaderView {
     pub random_value: CryptoHash,
     pub validator_proposals: Vec<ValidatorStakeView>,
     pub chunk_mask: Vec<bool>,
+    pub block_ordinal: NumBlocks,
```

Comment:
this should ideally also guarded by the feature flag. While not doing so has no impact on the protocol level, rpc users may get confused as to what this is.

Reply:
Updated.